package.json

```diff
@@ -18,20 +18,20 @@
     "generate-ci": "node .github/workflows/utils/generate.js"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^6.0.0",
```

Comment:
These are changes to the dependencies for *all* samples. If you think these should be updated for all samples, not just this one, can you please separate these out into a new PR with a description? Thanks!

Reply:
You're absolutely right! I thought I did need it in order for JSX linting to work, but I undid the changes to `package.json` and it looks like the linting still passes! The only global change that I do need in order for this Pull Request to pass linting is the change to `.eslintrc.json`.

Thank you!